MvvmCross/Binding/iOS/Target/MvxUITextViewTextTargetBinding.cs

```diff
@@ -46,7 +46,16 @@ public override void SubscribeToEvents()
                 return;
             }
 
-            target.Changed += this.EditTextOnChanged;
+			var textStorage = target.LayoutManager?.TextStorage;
+
+			if (textStorage == null)
+			{ 
+			    MvxBindingTrace.Trace(MvxTraceLevel.Error,
+						  "Error - NSTextStorage of UITextView is null in MvxUITextViewTextTargetBinding");
+				return;
+			}
+
+            textStorage.DidProcessEditing += this.EditTextOnChanged;
```

Comment:
I find it strange that it is still marked deprecated, but not fully in the first link I provided, which seems to be the correct documentation for this notification?

Although, creating a implementation in C# of NSTextStorageDelegate, it doesn't look like it is deprecated there.

🤔